db/models.py

```diff
@@ -40,13 +51,80 @@ def __str__(self) -> str:
 
 
 class MovieSession(models.Model):
-    show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
+    show_time: models.DateTimeField = models.DateTimeField()
+    cinema_hall: models.ForeignKey = models.ForeignKey(
         to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
     )
-    movie = models.ForeignKey(
+    movie: models.ForeignKey = models.ForeignKey(
         to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
```

Comment:
Ensure that the `AUTH_USER_MODEL` setting in your Django settings is configured to use this custom `User` model if it is intended to replace or extend the default user model.